neat/tools/fittools/compartmentfitter.py

```diff
@@ -196,8 +182,9 @@ def setImpedancesInTree(self, many_freqs=False, recompute=False, pprint=False):
         pprint: bool (optional, default is ``False``)
             Print info
         """
+        cname_string = '^'.join(list(self.channel_storage.keys()))
         if pprint:
-            print('>>> evaluating impedances with ' + str(list(self.channel_storage.keys())))
+            print('>>> evaluating impedances with %s'%cname_string)
```

Comment:
```suggestion
            print(f'>>> evaluating impedances with {cname_string}')
```
if we are allowed to use f-strings ;)